### PR TITLE
update wp and plugins

### PR DESCRIPTION
**Pull request by stephdavid**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,6 +25,14 @@ run:
...
 	docker-compose exec wordpress bash
 
+# from within docker; run a db import on the first .sql file found in the current directory and add an admin user
+db:
+	bin/local-db-import.sh
+
+# from within docker; run a db import on the first .sql file found in the current directory and add an admin user
+db:
+	bin/local-db-import.sh
+
```
Comment (wilson1000-MoJ) [inline Makefile:35]: We're you able to run `make db` ? There are 2 entries here - it would be best to remove one :)

**Comment by stephdavid** [comment]
Hello Damien, Would you please review and approve this update.  Thanks.

**Comment by stephdavid** [comment]
I have added a second commit "change email on dashboard and update wp and plugins" to this branch and pull request.

**Review by wilson1000-MoJ — COMMENTED** [review]
Hi Stephanie,

I have made some adjustments to the base Dockerfile and included some of your updates in a new PR. Also, you have WASM now so I am deleting the `local-db-import.sh` you don't need this; `wasm migrate <site>:<deploy> .` replaces the functionality.

I'll put that up for review and close this one.

**Pull request closed without merging**